Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -596,6 +596,38 @@ class RUMViewScopeTests: XCTestCase {
         XCTAssertEqual(viewUpdate.model.view.error.count, 1)
     }
 
+    func testGivenStartedView_whenCrossPlatformErrorIsAdded_itSendsCorrectErrorEvent() throws {
+        var currentTime: Date = .mockDecember15th2019At10AMUTC()
+        let isCrash: Bool = .random()
```

Comment:
i think we can make this `true`, it should be good enough if `true -> true` works (no need to test `false -> false` too after that)